.github/workflows/npm-audit-fix.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        branches: ["main", "master", "stable26", "stable25", "stable24"]
+        branches: ["main", "master", "stable27", "stable26", "stable25", "stable24"]
```

Comment:
As Nextcloud 24 reached its end of life, should `stable24` have been removed from there? Or was it kept for enterprise editions or something like that?

Reply:
> Or was it kept for enterprise editions or something like that?

Yes it was

Reply:
Thanks for the confirmation.